tokio-console/src/main.rs

```diff
@@ -68,7 +71,15 @@ async fn main() -> color_eyre::Result<()> {
             warnings::Linter::new(warnings::NeverYielded::default()),
         ])
         .with_retain_for(retain_for);
-    let mut input = input::EventStream::new();
+    let mut input = Box::pin(input::EventStream::new().try_filter(|event| {
+        ready(!matches!(
+            event,
+            input::Event::Key(input::KeyEvent {
+                kind: input::KeyEventKind::Release,
+                ..
+            })
```

Comment:
style nit, not important: it's up to you, but i might import `Event`, `KeyEvent`, and `KeyEventKind` so we can drop the `input::` prefix here and make the match pattern here a bit less wordy.